dapr_agents/workflow/agentic.py

```diff
@@ -128,8 +105,12 @@ def model_post_init(self, __context: Any) -> None:
         """
         self._dapr_client = DaprClient()
         self._text_formatter = ColorTextFormatter()
-        self._state_store_client = DaprStateStore(store_name=self.state_store_name)
-        logger.info(f"State store '{self.state_store_name}' initialized.")
+
+        # Set storage key based on agent name
+        self.storage._set_key(self.name)
```

Comment:
all internal storage attributes should be set during instantiation, or via public api methods, let's try to avoid direct internal method access (ofc be pragmatic but let's not abused them)